pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -35,14 +38,118 @@ func testStableSystemOperatorStateTransitions(events monitorapi.Intervals) []*ju
 			}
 		}
 
-		return "We are not worried about Available=False or Degraded=True blips for stable-system tests yet.", nil
+		isSingleNode, err := isSingleNodeCheck(clientConfig)
+		if err != nil {
+			logrus.Warnf("Error checking for Single Node configuration on stable system (unable to make exception): %v", err)
+			isSingleNode = false
+		}
+
+		// For the non-upgrade case, if any operator has Available=False, fail the test.
+		if condition.Type == configv1.OperatorAvailable {
+
+			// We'll add an exception for single node for now.
```

Comment:
Confusing comment I think, says we're adding an exception but we don't actually return the exception, we return empty string and nil which appears to actually mean no exception. I think we then trigger the degraded=True message below as the exception, which isn't even accurate for Available=false. 

I would suggest something like:

```
if condition.Type == configv1.OperatorAvailable && condition.Status == configv1.ConditionFalse {

			// We'll add an exception for single node for now.
                        if isSingleNode {
                               return "Operators are allowed to go degraded on single-node for now", nil 
                       }
			return "", nil
		}
```